docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1419,6 +1419,27 @@ rpc-ws-port="6174"
 Specifies Websockets JSON-RPC listening port (TCP).
 The default is 8546. Ports must be [exposed appropriately](../../HowTo/Find-and-Connect/Configuring-Ports.md).
 
+### target-gas-limit
+
+```bash tab="Syntax"
+--target-gas-limit=<INTEGER>
+```
+
+```bash tab="Command Line"
+--target-gas-limit=nnn
```

Comment:
Rather than `nnn` this needs to be an actual example - the point of including is so the user can look at this and see a specific example of the format.  eg, when I tried to start a node with this CLI initially I assumed the value would be in hex but it's decimal.  An example would have saved me going down the wrong path `--target-gas-limit=nnn`

Reply:
Added realistic example